ChangeLog.md

```diff
@@ -27,6 +27,10 @@ See docs/process.md for more on how version tagging works.
   `warning: treating 'c' input as 'c++' when in C++ mode`.  This also means that
   the `DEFAULT_TO_CXX` setting now only applies when linking and not when
   compiling. (#20712)
+- JavaScript library code can now use the full range ES6 features and we rely
+  on closure compiler to transpile for ES5 when targetting older browsers.
+  For those that want to would rather perform transpilation seperately outside
+  of emscripten you can use the `-sPOLYFILL=0` setting.  (#20700)
```

Comment:
What is the connection between transpiling and polyfills? That `POLYFILLS` disables transpiling seems slightly surprising to me and I think might be for other readers.

Reply:
Yes, I kind of lumped them together.   

I do think its somewhat logical though since folks who use `-sPOLYFILL=0` are basically saying "I promise to install my own polyfills".. otherwise the code simply won't run, and in almost all cases that I know of polyfills are normally installed by transpilers, so what they are really saying is "I promise to my own transpilation" to ES5 (or whatever).

I've not actually heard of anyone using `-sPOLYFILL=0` in the wild, but if we hear back from suchusers that they want to use this setting *and* still have emscripten take care of transpiling we can consider a separate setting.